ironfish/src/rpc/adapters/httpAdapter.ts

```diff
@@ -185,6 +197,10 @@ export class RpcHttpAdapter implements IRpcAdapter {
     }
 
     const combined = Buffer.concat(data)
+
+    this.inboundTraffic.add(size)
+    this.inboundBytes.value += size
```

Comment:
this makes me wonder if we should expose `Meter._count` so that we wouldn't need a separate `Gauge` to track the total added to the `Meter` 🤔

Reply:
Prometheus does expose a specific metric type for counting values that only go up: https://prometheus.io/docs/concepts/metric_types/